ga4gh/protocol.py

```diff
@@ -26,6 +26,17 @@ def convertDatetime(t):
     return int(millis)
 
 
+class ProtocolException(Exception):
+    """
+    Something went wrong in the protocol module
+    """
+
+
+class UnableToDecodeJsonException(Exception):
```

Comment:
It's not necessarily a server error.  In fact both of these were written with client errors in mind.  Should they go in `exceptions.py` (and just not inherit from BaseServerException) or in a separate module?